src/pages/import/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { useState } from 'react';
+
+import {
+  Spinner,
+  VStack,
+  Heading,
+  useMediaQuery,
+  HStack,
+  Flex,
+  Box,
+  ButtonGroup,
+  Button,
+  Center,
+} from '@chakra-ui/react';
+import { useParams } from 'react-router';
+
+import { Timetable, useGetTimetable } from 'lib/fetchers';
+import { getReadableTerm } from 'lib/utils/terms';
+
+import { Page } from 'common/layouts/Page';
+import { TopBar } from 'common/layouts/sidebar/components/TopBar';
+import { Sidebar } from 'common/layouts/sidebar/containers/Sidebar';
+
+import { ImportCalendar } from './components/ImportCalendar';
+import { TimetableActionButtons } from './components/TimetableActionButtons';
+import { TimetableCourseCard } from './components/TimetableCourseCard';
+
+export function ImportTimetable(): JSX.Element {
+  const { slug } = useParams();
+  const [isMobile] = useMediaQuery('(max-width: 1020px)');
+
+  const { loading, data } = useGetTimetable({ slug: slug });
+
+  const left = (
+    <>
+      <TopBar>Timetable Actions</TopBar>
+      <TimetableActionButtons data={data as Timetable} loading={loading} />
+    </>
+  );
+
+  const right = (
+    <>
+      <TopBar>Included Courses</TopBar>
+      <Box h="100%" overflowY="auto" pb="20">
+        {!loading && data ? (
+          <VStack>
+            {(data as Timetable).courses.map((course) => (
+              <TimetableCourseCard course={course} term={(data as Timetable).term} />
+            ))}
+          </VStack>
+        ) : (
+          <></>
+        )}
```

Comment:
```suggestion
        {!loading && data && (
          <VStack>
            {(data as Timetable).courses.map((course) => (
              <TimetableCourseCard course={course} term={(data as Timetable).term} />
            ))}
          </VStack>
        )}
```
nit: don't need a ternary when the false option is `<></>`, this suggestion will only render the VStack if the condition is true which is same as what you're doing with the ternary, just a bit cleaner

Reply:
there's a few instances of this same thing that could be addressed eventually